assets/js/base/components/sidebar-layout/style.scss

```diff
@@ -50,6 +50,14 @@
 			padding-right: $gap;
 		}
 	}
+
+	.wc-block-components-sidebar {
+		.wc-block-components-panel {
```

Comment:
Minor: we'd probably want to move this into the `.is-large .wc-block-components-sidebar` block in line 46, no?